scripts/benchmark_dice.py

```diff
@@ -0,0 +1,67 @@
+from sklearn.compose import ColumnTransformer
+from sklearn.model_selection import train_test_split
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.ensemble import RandomForestClassifier
+import numpy as np
+import pandas as pd
+import time
+import dice_ml
+from dice_ml.utils import helpers  # helper functions
+from tensorflow import keras
+import tensorflow as tf
+
+
+df = pd.read_csv('assets/adult/data.csv')
+target = df["income"]
+
+continuous = ["age","hours_per_week"]
+categorical = ["workclass","education","marital_status","occupation","race","gender"]
+
+
+X_train, X_test, y_train, y_test = train_test_split(
```

Comment:
Write this:

```python
X_train, X_test, y_train, y_test = train_test_split(
                                                    df,
                                                    target,
                                                    test_size=0.2,
                                                    shuffle=False,
                                                    stratify=target)
```

Reply:
ValueError: Stratified train/test split is not implemented for shuffle=False